graphite/graphite.go

```diff
@@ -26,12 +26,30 @@ type Series struct {
 
 type DataPoint []json.Number
 
+type Error struct {
```

Comment:
actually thinking about it more, we shouldn't have 1 general purpose error clas. will rename to RequestErr and if we ever need other classes we can make em.